### ISSUE TITLE
Crash on click of datapoint in Insights - Sessions linear chart

### ISSUE THREAD

**Issue by stevenphaedonos** [PostHog/posthog#2950]
## Bug description

Clicking a datapoint on the line chart of the Insights - Sessions visualisation causes a crash. 

## Expected behavior

A modal should open with information related to the datapoint

## How to reproduce

1. Navigate to /insights?insight=SESSIONS
2. Click a datapoint on the linear linechart graph for Average Session Length

## Environment

- [ ] PostHog Cloud
- [x] self-hosted PostHog, version/commit: 650bc428

## Additional context

Here's the root-cause analysis:

- The response of `/api/insights/session/`, which provides the dataset for the visualisation, does not include an `action` parameter, unlike `/api/insights/trend/` 
- The datapoint onClick function calls `loadPeople` from `trendsLogic` as follows:
```
loadPeople(
  dataset.action || 'session',
  dataset.label,
  day,
  dataset.breakdown_value || dataset.status
)
```
- Given that there is no `dataset.action`, the fallback of **string** `session` is passed to `loadPeople`
- `loadPeople` calls `setPeople`, therefore updating `reducers.people` and hence `selectors.people` 
- `selectors.sessionsPageParams` is computed since it depends on `selectors.people`
- `selectors.sessionsPageParams` attempts to spread `selectors.people.action` which is a **string**, and therefore throws an error

## Resolution

Simply removing the spread operations against `selectors.people.action` resolves the issue, but I didn't have time to investigate the side-effects of this. I'll follow up with a fix once I have more free time!

**Comment by timgl** [comment]
Thanks for raising this! This was fixed in #2926, so would you mind trying to upgrade your instance and seeing if that did it? Feel free to re-open if you experience the issue again.

**Comment by ghost** [comment]
Yes, resolved, thanks!
